### DISCUSSION TITLE
Lets discuss how cool this is

### DISCUSSION THREAD

**Discussion by ghost** [dosemu2/dosemu2#1447] [General]
This is actually really cool

**Comment by stsp** [comment]
What do you want to discuss?
If there is something really cool,
you could add a screen-shot.

**Reply by ghost** [reply]
I just mean the concpet. DOSbox, is frankly not optimal, and wine cant run the msdos stuff

**Reply by stsp** [reply]
Please describe the concept that
you want to discuss. Write down
why exactly dosbox is not optimal
for you and how dosemu2 is better.
